[FEAT] 마이페이지 내프로필 조회/수정 API 구현 (#4)

Jang-Goeun opened the pull request:
# PR 템플릿

## #️⃣ 연관된 이슈

> #4



## 체크리스트!
- [x] 🔀 PR 제목의 형식을 잘 작성했나요?
- [x] 🧹 불필요한 코드는 제거했나요?
- [x] 💭 이슈는 등록했나요?
- [x] 🏷️ 라벨은 등록했나요?



## 📝작업 내용

> 마이페이지 내프로필 조회/수정 api 구현



### PR 특이 사항

> 기본 틀만 작성된 점 참고 부탁드립니다!!

Jang-Goeun committed:
Refac: 마이페이지 내프로필 관련 엔티티 클래스 수정 (#4)

연관관계 추가, 필요 어노테이션 추가

Jang-Goeun committed:
Feat : 마이페이지 내프로필 수정 기능 추가 (#4)

Jang-Goeun committed:
Feat : 마이페이지 내프로필 조회 기능 추가 (#4)

Jang-Goeun committed:
Merge branch 'develop' into feat/mypageInfo/#4

dokyung-kang commented on src/main/java/com/partnerd/repository/MemberRepository.java:
```diff
@@ -0,0 +1,7 @@
+package com.partnerd.repository;
```
memberRepository패키지 생성해서 안에 넣어주세요!

dokyung-kang commented on src/main/java/com/partnerd/repository/AgreementsRepository.java:
```diff
@@ -0,0 +1,7 @@
+package com.partnerd.repository;
```
agreementsRepository패키지 안으로 옮겨주세요!

dokyung-kang commented on src/main/java/com/partnerd/web/controller/MemberRestController.java:
```diff
@@ -0,0 +1,55 @@
+    // 내프로필 조회
+    @GetMapping("/users/me/info/{memberId}")
+    @Operation(summary = "내프로필 조회 API",description = "마이페이지의 내프로필 정보를 조회하는 API입니다.")
+    @ApiResponses({
```
COMMON200으로 통일시켜주세요!

Jang-Goeun committed:
Remove : MemberRepository 파일 삭제 (#4)

중복된 파일 삭제함

Jang-Goeun committed:
Rename : AgreementsRepository 파일 옮김 (#4)

AgreementsRepository 패키지 하위로 파일 옮김

Jang-Goeun committed:
Refac : 마이페이지 내프로필 관련 코드 수정 (#4)

Member 파일 공백 제거, MemberServiceImpl import문 변경, MemberRestController apiResponses 수정
